src/azul/logging.py

```diff
@@ -6,8 +6,43 @@
 )
 
 
+# lambda python bootstrap.py line 284 - to add format attribute of aws_request_id
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, aws_request_id):
+        super().__init__()
+        self.aws_request_id = aws_request_id
+
+    def filter(self, record):
+        try:
+            record.aws_request_id = _GLOBAL_AWS_REQUEST_ID or ""
```

Comment:
I think we can remove the `_GLOBAL_AWS_REQUEST_ID` and always use the self.aws_rest_id to populate the record.aws_request_id